src/applications/travel-pay/services/mocks/index.js

```diff
@@ -0,0 +1,11 @@
+const delay = require('mocker-api/lib/delay');
+
+const commonResponses = require('../../../../platform/testing/local-dev-mock-api/common');
+const travelClaims = require('./travel-claims.json');
+
+const responses = {
+  ...commonResponses,
+  'GET /v0/travel-claims': travelClaims,
+};
+
+module.exports = delay(responses, 1000);
```

Comment:
How is this file used?

Reply:
This file allow for a mocked API to run without have to run the vets-api. 

reference: https://github.com/department-of-veterans-affairs/vets-website?tab=readme-ov-file#running-a-mock-api-for-local-development

Reply:
Oh thanks, I've never had the chance to use that. TIL!